src/Uno.UWP/UI/Composition/CompositionSpriteShape.Skia.cs

```diff
@@ -39,11 +36,15 @@ internal override void Render(SKSurface surface, SKImageInfo info)
 						if (StrokeBrush is CompositionColorBrush stroke)
 						{
 							_strokePaint.StrokeWidth = StrokeThickness;
-							_strokePaint.Color = stroke.Color.ToSKColor();
+							_strokePaint.Color = stroke.Color.ToSKColor(Compositor.CurrentOpacity);
 						}
 
 						surface.Canvas.DrawPath(geometrySource.Geometry, _strokePaint);
 					}
+				}
+				else if(cpg.Path.GeometrySource is null)
+				{
```

Comment:
Codacy found an issue: [Either remove or fill this block of code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5924508)